cmd/Kconfig

```diff
@@ -1490,6 +1490,13 @@ config CMD_MDIO
 	  The MDIO interface is orthogonal to the MII interface and extends
 	  it by adding access to more registers through indirect addressing.
 
+config CMD_WGET
+	bool "wget"
+	select TCP
+	help
+	  Download kernel, or other files, from a web server over TCP.
+	  Fast file transfer over networks with latenc
```

Comment:
Is that my typo or inherited from the v2016 release?

Reply:
It isn't present in the repo that appears to be the origin of this implementation, https://github.com/ProfFan/u-boot/commit/d5a3abf08f1a5c8c29c1521fa141652a3e39a515

It appears to have originated in:
e023532 2018-12-18 Added TCP/wget implementation

Reply:
Oops.

I don't really prefer either way if this gets corrected; that's up to you if you want to fix and squash. We generally don't use menu/config for U-Boot so this output would likely never really be seen.